code/game/objects/items/explosives/grenades/sticky_grenades.dm

```diff
@@ -101,34 +73,19 @@
 	det_time = 5 SECONDS
 	light_impact_range = 1
 	self_sticky = TRUE
-	/// smoke type created when the grenade is primed
-	var/datum/effect_system/smoke_spread/smoketype = /datum/effect_system/smoke_spread/tactical
-	///radius this smoke grenade will encompass
-	var/smokeradius = 1
-	///The duration of the smoke
-	var/smoke_duration = 8
```

Comment:
if someone adds a new smoke type they're gonna have to undo these changes but eh, future problem I guess?